docs/src/lib/sets/Zonotope.md

```diff
@@ -18,7 +18,9 @@ split(::AbstractZonotope, ::Int)
 split(::AbstractZonotope, ::AbstractVector{Int}, ::AbstractVector{Int})
 remove_zero_generators(::Zonotope)
 linear_map!(::Zonotope, ::AbstractMatrix, ::Zonotope)
-quadratic_map(Q::Vector{MT}, Z::Zonotope{N}) where {N, MT<:AbstractMatrix{N}}
+quadratic_map(::Vector{MT}, ::Zonotope{N}) where {N, MT<:AbstractMatrix{N}}
+LazySets._overapproximate_zonotope_vrep
+LazySets._overapproximate_zonotope_cpa
```

Comment:
These must go to `approximations.md` and need `Approximations` as prefix instead.